lua/plenary/popup.lua

```diff
@@ -29,16 +29,16 @@ local function dict_default(options, key, default)
 end
 
 
-function popup.popup_create(what, vim_options)
-  local buf
+function popup.popup_create(what, options)
```

Comment:
this should not have changed from `vim_options` btw. It is the table as describe in the vim help docs, (not neovim) so it has semantic meaning.

Reply:
Oops, sorry, don't know why I changed that! Fixed.